route_info/src/map_matching.rs

```diff
@@ -27,55 +25,4 @@ impl RouteInfo {
             bail!("not a LineString");
         }
     }
-
-    /// Interpret waypoints from route-snapper in the context of this StreetNetwork.
```

Comment:
Some of the code could be useful in some unknown future, but git means we lose nothing, and it's unused in the meantime